### PR TITLE
Mara/reviewer's half of the visdat 📦 review post

### PR DESCRIPTION
**Pull request by batpigandme**
## Blog post?
My blog post for the package review of `visdat`. The dummy link for getting to Nick's post is on line 39 (which it won't let me link to for some reason).

All feedback, edits, etc. welcome… 😳

### CODE REVIEW

File: _posts/2017-08-23-first-package-review.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Myriad lessons learned 💡
+
+I can't speak for Nick per the utility or lack thereof of my review (you can see [his take here](URL)), but I _can_ vouch for the package-reviewing experience as a means of methodically inspecting the innards of an R package. Methodical is really the operative word here. Though _"read the docs,"_ or _"look at the code"_ sounds straight-forward enough, it's not always easy to coax oneself into going through the task piece-by-piece without an end goal in mind. While a desire to contribute to open-source software is noble enough (and is how I _personaly_ ended up involved in this process-- with some help/coaxing from [Noam Ross](https://twitter.com/noamross)), it's also an abstraction that can leave one feeling overwhelmed, and not knowing where to begin.[^3]
```
Comment (batpigandme) [inline _posts/2017-08-23-first-package-review.md]: placeholder link here

**Comment by stefaniebutland** [comment]
Thanks for doing this @batpigandme !! Will make a great pair of posts.
Because many people will read your post independently of Nick's, can you please add a line at the start saying that this is about doing your first pkg review for rOpenSci or something to clearly set the context? You say it later, but I don't think it'll be clear enough to naive reader

**Comment by batpigandme** [comment]
@stefaniebutland Tried to make it a bit clearer [see here](https://github.com/ropensci/roweb/pull/335/commits/0f046d316fb1d81041b356567d72d00d03fc9f6a) I changed the beginning of that first sentence from my earlier, but I see what you mean re. the entire topic of what I was doing going unmentioned… 🙈

**Comment by sckott** [comment]
looks good @batpigandme - thanks much!

**Comment by stefaniebutland** [comment]
This is GREAT @batpigandme. The new line introducing the post is parfait!

**Pull request merged**